### PR TITLE
Seismic misfit improvements

### PR DESCRIPTION
**Pull request by rnyb**
*General improvements*

---

### Contributor checklist

- [ ] :tada: This PR closes #ISSUE_NUMBER.
- [x] :scroll: I have broken down my PR into the following tasks:
   - [x] Reworked all colors. Use same symmetric colorscale for all types of attributes. Also new color scales for diff plot and error plot. Colorscales are included as global variables.
   - [x] Prettyfied all hoverdata
   - [x] New option for slice plot. show individual realizations
   - [x] Keep user defined zoom during callback
- [ ] :robot: I have added tests, or extended existing tests, to cover any new features or bugs fixed in this PR.
- [x] :book: I have considered adding a new entry in `CHANGELOG.md`, and added it if should be communicated there.

### CODE REVIEW

File: webviz_subsurface/plugins/_seismic_misfit.py [hunk 1]
```diff
@@ -3405,7 +3520,9 @@ def make_polygon_df(ensemble_set: dict, polygon: str) -> pd.DataFrame:
                             "(default is None)"
                         )
                     else:
-                        df_polygon["name"] = str(Path(poly_file).name)
+                        name = str(Path(poly_file).name)
+                        name = name.replace(".csv", "").replace("_", " ")
+                        df_polygon["name"] = name
```
Comment (anders-kiaer) [inline webviz_subsurface/plugins/_seismic_misfit.py:3525]: Maybe you can use (needs to be tested):
```suggestion
                        df_polygon["name"] = Path(poly_file).stem.replace("_", " ")
```

**Review by anders-kiaer — CHANGES_REQUESTED** [review]
One comment - otherwise LGTM.

File: webviz_subsurface/plugins/_seismic_misfit.py (hunk 1 above)
Comment (rnyb) [reply webviz_subsurface/plugins/_seismic_misfit.py:3525]: Nice one, will fix.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [UNRELEASED] - YYYY-MM-DD
 
+### Added
```
Comment (anders-kiaer) [inline CHANGELOG.md:9]: Does the change belong more under the `Changed` category?

**Review by anders-kiaer — APPROVED** [review]

**Pull request merged**